docs/app/pages/components/components-sections/input-controls/tags/tags.component.html

```diff
@@ -43,15 +43,18 @@
         <ux-accordion>
             <ux-accordion-panel class="accordion-chevron" heading="Customize Example...">
                 <div class="row uxd-customize-row">
-                    <div class="col-md-4 col-sm-12">
+                    <div class="col-md-3 col-sm-12">
```

Comment:
You will notice there is a `snippets` folder in here, the code will need to be updated in there too to ensure the plunker and sample code has the same functionality as this.

Reply:
Added updates to plunker